libs/wallet/src/vega-transaction-dialog/vega-transaction-dialog.tsx

```diff
@@ -159,108 +98,77 @@ export const VegaDialog = ({
             </a>
-          </p>
-        )}
-      </OrderDialogWrapper>
-    );
-  }
-
-  // Order on network but was rejected
-  if (finalizedOrder.status === 'Rejected') {
-    return (
-      <OrderDialogWrapper
-        title="Order failed"
-        icon={<Icon name="warning-sign" size={20} />}
-      >
-        <p data-testid="error-reason">
-          {finalizedOrder.rejectionReason &&
-            t(`Reason: ${formatLabel(finalizedOrder.rejectionReason)}`)}
+          )}
         </p>
-      </OrderDialogWrapper>
+      </div>
     );
   }
 
-  return (
-    <OrderDialogWrapper title={title} icon={<Icon name="tick" size={20} />}>
-      <div className="grid grid-cols-1 md:grid-cols-2 gap-8">
-        {finalizedOrder.market && (
-          <div>
-            <p className={headerClassName}>{t(`Market`)}</p>
-            <p>{t(`${finalizedOrder.market.name}`)}</p>
-          </div>
-        )}
-        <div>
-          <p className={headerClassName}>{t(`Status`)}</p>
-          <p>{t(`${finalizedOrder.status}`)}</p>
-        </div>
-        {finalizedOrder.type === OrderType.Limit && finalizedOrder.market && (
-          <div>
-            <p className={headerClassName}>{t(`Price`)}</p>
-            <p>
-              {addDecimalsFormatNumber(
-                finalizedOrder.price,
-                finalizedOrder.market.decimalPlaces
-              )}
-            </p>
-          </div>
-        )}
-        <div>
-          <p className={headerClassName}>{t(`Amount`)}</p>
-          <p
-            className={
-              finalizedOrder.side === 'Buy'
-                ? 'text-vega-green'
-                : 'text-vega-red'
-            }
-          >
-            {`${finalizedOrder.side === 'Buy' ? '+' : '-'} ${
-              finalizedOrder.size
-            }
-            `}
-          </p>
-        </div>
-      </div>
-      <div className="grid grid-cols-1 gap-8">
-        {transaction.txHash && (
-          <div>
-            <p className={headerClassName}>{t(`Transaction`)}</p>
+  if (transaction.status === VegaTxStatus.Complete) {
+    return (
+      <div data-testid={transaction.status}>
+        <p className="break-all">
+          {t('Your transaction has been confirmed')} - &nbsp;
+          {transaction.txHash && (
             <a
-              className="underline break-words"
+              className="underline"
               data-testid="tx-block-explorer"
               href={`${VEGA_EXPLORER_URL}/txs/0x${transaction.txHash}`}
               target="_blank"
               rel="noreferrer"
             >
-              {transaction.txHash}
+              {t('View in block explorer')}
             </a>
-          </div>
-        )}
+          )}
+        </p>
       </div>
-    </OrderDialogWrapper>
-  );
+    );
+  }
+
+  return null;
 };
 
-interface OrderDialogWrapperProps {
-  children: ReactNode;
-  icon: ReactNode;
-  title: string;
-}
+const getIntent = (transaction: VegaTxState) => {
```

Comment:
(blocking): pulled the branch and tested. there is still sth not working correctly yet, the order has failed and we have a rejection reason but still displaying successful intent of the dialog

<img width="1229" alt="Screenshot 2022-07-23 at 10 05 40" src="https://user-images.githubusercontent.com/16125548/180596958-e67bb035-6d37-4f05-a25e-5445aea7aa37.png">